Android/app/src/main/java/app/intra/DnsVpnAdapter.java

```diff
@@ -60,30 +69,140 @@
 
   private static final int DNS_DEFAULT_PORT = 53;
 
+  // VPN parameters
+  private static final int VPN_INTERFACE_MTU = 32767;
```

Comment:
You could extract this variable to `VpnAdapter`, since it is also used by `SocksVpnAdapter`.

Reply:
Done